examples/tree_1d_dgsem/elixir_advection_diffusion.jl

```diff
@@ -37,11 +39,11 @@ function initial_condition_diffusive_convergence_test(x, t,
     x_trans = x_trans_periodic(x - equation.advection_velocity * t)
 
     nu = diffusivity()
-    c = 0.0
-    A = 1.0
+    c = 0
+    A = 1
     L = 2
-    f = 1 / L
-    omega = 1.0
+    f = 1.0f0 / L
```

Comment:
Is there any reason for keeping this `f` @ranocha ?

Reply:
It does not seem to be used anywhere as far as I see. Thus, we can remove it and everything just defined for it, e.g., `L`.

```suggestion
```